### ISSUE TITLE
autosize=false blocks the opening of the menu after being focused

### ISSUE THREAD

**Issue by geraldodev** [JedWatson/react-select#1316]
Hi,

When autosize=false the code on handleMouseDown blocks the menu of being opened for the second time after getting the focus. You will not be able to open it by clicking. When you focus anywhere else and click back the menu opens again. This is does not affect  AutosizeInput.

```
        if (event.target.tagName === 'INPUT') {
            return;
        }
```

regards,
geraldodev

**Comment by dave-clover** [comment]
I'm also having trouble with this line of code.  It's breaking our change that allows you to right click and paste into the select component.

@gojohnnygo was the original author of that code.  Would you mind elaborating on what issue it is fixing?

**Comment by gojohnnygo** [comment]
@dave-clover can you link me the line in question?

**Comment by dave-clover** [comment]
Hey @gojohnnygo, thanks for the quick response.  This line:

https://github.com/JedWatson/react-select/blob/master/src/Select.js#L322

We're probably going to have to make a change to fix the original issue in this thread, but we don't want to break the thing you were fixing.

**Comment by gojohnnygo** [comment]
Hey @dave-clover,

That fixed a minor UI quirk. Here's the PR describing what it does. https://github.com/JedWatson/react-select/pull/946

**Comment by cbergmiller** [comment]
I have submitted a PR that should solve this issue: #1746

**Comment by JedWatson** [comment]
I've been able to reproduce this and resolved it differently to #1887, see the referenced commit for more information
